appinventor/buildserver/src/com/google/appinventor/buildserver/AabCompiler.java

```diff
@@ -0,0 +1,314 @@
+package com.google.appinventor.buildserver;
+
+import com.google.common.io.Files;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+/**
+ * This Callable class will convert the compiled files into an Android App Bundle.
+ * An AAB file structure looks like this:
+ * - assets.pb
+ * - resources.pb
+ * - native.pb
+ * - manifest/AndroidManifest.xml
+ * - dex/
+ * - res/
+ * - assets/
+ * - lib/
+ */
+public class AabCompiler implements Callable<Boolean> {
+  private PrintStream out;
+  private File buildDir;
+  private int mx;
+
+  private AabPaths aab;
+  private String originalDexDir = null;
+  private File originalLibsDir = null;
+
+  private String bundletool = null;
+  private String jarsigner = null;
+
+  private String deploy = null;
+  private String keystore = null;
+
+  private class AabPaths {
+    private File ROOT = null;
+    private File BASE = null;
```

Comment:
Any reason for these to be all caps? That is not really the naming convention used in the rest of the code. Please make them lower case and update the names of the corresponding accessors.

Reply:
It was because they were like the "main directories". But changed it back to lowercase.